nilearn/maskers/nifti_labels_masker.py

```diff
@@ -22,12 +22,14 @@ def __init__(self, _resampled_labels_img_, background_label, strategy,
         self.mask_img = mask_img
 
     def __call__(self, imgs):
-        from ..regions import signal_extraction
+        from ..regions.signal_extraction import img_to_signals_labels
 
-        return signal_extraction.img_to_signals_labels(
+        signals, labels, masked_labels_img = img_to_signals_labels(
             imgs, self._resampled_labels_img_,
             background_label=self.background_label, strategy=self.strategy,
-            keep_masked_labels=self.keep_masked_labels, mask_img=self.mask_img)
+            keep_masked_labels=self.keep_masked_labels, mask_img=self.mask_img,
+            return_masked_atlas=True)
+        return signals, (labels, masked_labels_img)
```

Comment:
Thx for handling that !

This is a breaking change of the API... we need to go through deprecation cycles.

Reply:
Is it because of the return? Could you explain a bit more? I thought it should be fine.

Reply:
If I'm not mitsaken, we move from 2 to 3 return arguments.

Reply:
I put them in parentheses so I think it will still count as two arguments. It was @jeromedockes idea. and it didn't fail any tests. Maybe I'm missing something?

Reply:
But it can break code from third parties.  
To be honest, I don't have a better suggestion, but I'm wondering whether we should not use explicit deprecation cycles.

Reply:
@bthirion I see. You mean people who use this private function assume that the second output is just `labels` and that will cause errors?

Reply:
Yes, exactly.